src/shred_network/repair_service.zig

```diff
@@ -52,11 +53,11 @@ pub const RepairService = struct {
     shred_tracker: *BasicShredTracker,
     logger: ScopedLogger(@typeName(Self)),
     exit: *Atomic(bool),
-    last_big_request_timestamp_ms: i64 = 0,
     /// memory to re-use across iterations. initialized to empty
     report: MultiSlotReport,
     thread_pool: RequestBatchThreadPool,
     metrics: Metrics,
+    rng: std.Random.DefaultPrng,
```

Comment:
i think the convention is to name this `prng` -- @InKryption is this correct? do we have a link to where we defined the random rules?

Reply:
ba1baafae1c09cefc295c08a238dce1828f697b5